File: javascript/problem1.js
const sortCharacters = (s) => {
  const vowels = "aeiou";
  let vowelChars = "";
  let consonantChars = "";
  s = s.toLowerCase();
  for (let char of s) {
    if (char === " ") {
      continue;
    }
    if (vowels.includes(char)) {
      vowelChars += char;
    } else if (char >= "a" && char <= "z") {
      consonantChars += char;
    }
  }
  return { vowelChars, consonantChars };
};

const input = "Sample Case";
const { vowelChars, consonantChars } = sortCharacters(input);
console.log("Vowel Characters:", vowelChars);
console.log("Consonant Characters:", consonantChars);

File: golang/problem1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strings"
)

func sortCharacters(s string) (string, string) {
	vowels := "aiueo"
	var vowelsCars, consonanChart string

	s = strings.ToLower(s)
	for _, char := range s {
		if strings.ContainsRune(vowels, char) {
			vowelsCars += string(char)
		} else if char >= 'a' && char <= 'z' {
			consonanChart += string(char)
		}
	}

	vowelsSlice := strings.Split(vowelsCars, "")
	consonantsSlice := strings.Split(consonanChart, "")
	sort.Strings(vowelsSlice)
	sort.Strings(consonantsSlice)

	return strings.Join(vowelsSlice, ""), strings.Join(consonantsSlice, "")
}

func main() {
	reader := bufio.NewReader(os.Stdin)
	fmt.Print("Enter a string: ")
	input, _ := reader.ReadString('\n')
	input = strings.TrimSpace(input)

	vowels, consonants := sortCharacters(input)
	fmt.Println("Vowel Characters:", vowels)
	fmt.Println("Consonant Characters:", consonants)
}
